lib/import-from-tiled.js

```diff
@@ -0,0 +1,135 @@
+				image_index = layer.data[j] - file.tilesets[tileset_index].firstgid;
+				tileset = file.tilesets[tileset_index];
+
+				if (image_index >= 0) {
+
+					// Create tile and get properties
+					tile = entities.create();
+					entities.set(tile, "name", "tile");
+					entities.set(tile, "tile", true);
+					entities.set(tile, "image", { "name": "" });
+					entities.set(tile, "position", { "x": 0, "y": 0 });
+					entities.set(tile, "size", { "width": tileset.tileheight, "height": tileset.tilewidth });
+					image = entities.get(tile, "image");
+					tile_pos = entities.get(tile, "position");
+
+					// Position based on index
+					tile_pos.x = (j % file.width) * file.tilewidth;
+					tile_pos.y = Math.floor(j / file.width) * file.tileheight;
+
+					// Character position z: 1 so anything without background true will layer over player
+					tile_pos.z = (layer.properties.Background == "True") ? -1 : 2;
+
+					// Select which "tile" of the tileset image to render
+					image.name = tileset.image;
+					image.sourceWidth = tileset.tilewidth;
+					image.sourceHeight = tileset.tileheight;
+					image.destinationWidth = tileset.tilewidth;
+					image.destinationHeight = tileset.tileheight;
+					image.sourceX = (image_index % cols) * tileset.tilewidth + ((image_index % cols) * tileset.spacing) + tileset.margin;
+					if(tileset.tileheight == tileset.imageheight) {
+						image.sourceY = 0;
+					} else {
+						image.sourceY = Math.floor(image_index / cols) * tileset.tileheight + (Math.floor(image_index / cols) * tileset.spacing) + tileset.margin;
+					}
+
+				}
+			}
+		}
+
+		/**
+		 * @member {Entity} Object Entities
+		 * <p>Create Entity with size and position components</p>
+		 * <p><b>Collidable</b>: True or False <br />Add collision component to entity.
+		 * Currently collisions are handled as a simulation system in the project.</p>
+		 * <p><b>Spawn</b>: True or False <br />Creates an entity with a "spawn" boolean
+		 * component allowing you to use <code>game.entities.find("spawn");</code> to find the spawn point for this map.</p>
+		 * <br />All other properties end up in a "tiledProperties" component to be handled on a per-game basis.
+		 */
+		// Loop object layer for collisions and spawn point
+		if (layer.type == "objectgroup") {
+			for (var j = 0; j < layer.objects.length; j++) {
+				object = layer.objects[j];
+				object_properties = {};
+				entity = entities.create();
+				entities.set(entity, "size", { "width": object.width, "height": object.height });
+				entities.set(entity, "position", { "x": object.x, "y": object.y });
+				entities.set(entity, "collisions", []);
+				Object.keys(object.properties).forEach(function(key) {
+					if (!isNaN(object.properties[key])) {
+						object_properties[key] = parseInt(object.properties[key], 10);
+					}
+					if (new String(object.properties[key]).toLowerCase() == "true" || object.properties[key] == "1") {
```

Comment:
It's probably a bad idea to coerce `"0"` to `false` and `"1"` to `true`. I think this will also coerce 0 and 1 created by the integer check above. It's probably best to only do "true" and "false".